tests/ai_hub/model_catalog/plugin_arch/test_unified_catalog_server.py

```
@@ -28,41 +28,55 @@ def test_catalog_pod_initializes_plugins(
-        expect_plugins: bool,
+        plugin_name: str,
     ):
         """Given the catalog server is running with registered plugins
-        When querying a health endpoint
-        Then the server reports the expected status
+        When querying the readyz endpoint
+        Then the plugin is registered and healthy
         """
         response = requests.get(
-            f"{catalog_base_url}/{endpoint}", headers=model_registry_rest_headers, verify=False, timeout=30
+            f"{catalog_base_url}/readyz", headers=model_registry_rest_headers, verify=False, timeout=30
```

Ruff (0.15.15)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)